rma_sale/views/sale_portal_template.xml

```diff
@@ -32,18 +32,17 @@
                             </div>
                             <t
                                 t-set="delivery_addresses"
-                                t-value="sale_order.partner_id.commercial_partner_id.mapped('child_ids').filtered(lambda x: x.type in ['contact', 'delivery'])"
+                                t-value="sale_order.partner_shipping_id | sale_order.partner_id.commercial_partner_id.mapped('child_ids').filtered(lambda x: x.type in ['contact', 'delivery'])"
                             />
                             <button
                                 class="btn btn-primary btn-block mb8"
                                 type="button" data-toggle="collapse"
                                 data-target="#delivery_address_picker"
                                 aria-expanded="false"
-
                             ><i class="fa fa-truck" /> Choose a delivery address</button>
                             <div class="col-lg-12 collapse mt8" id="delivery_address_picker">
                                 <div data-toggle="buttons" class="row">
-                                    <label t-attf-class="card mr4 btn btn-light" t-foreach="delivery_addresses" t-as="address">
+                                    <label t-attf-class="card mr4 btn btn-light #{address == sale_order.partner_shipping_id and 'active' or ''}" t-foreach="delivery_addresses" t-as="address">
```

Comment:
Shouldn't this be the previous selected address?

```suggestion
                                    <label t-attf-class="card mr4 btn btn-light #{address == delivery_addresses and 'active' or ''}" t-foreach="delivery_addresses" t-as="address">
```

Reply:
No, for the reason exposed before